lib/hooks.rb

```diff
@@ -1,17 +1,23 @@
 
 class Hooks
-  def self.run(kind, failover_ip, old_ip, new_ip)
+  def self.run(kind, failover_ip, old_ip, new_ip, dry = false)
     Dir[File.expand_path("../../hooks/#{kind}/*", __FILE__)].sort.each do |file|
-      system(file, failover_ip, old_ip, new_ip) if File.executable?(file)
+      if File.executable?(file)
+        if !dry
+          system(file, failover_ip, old_ip, new_ip)
+        else
+          $logger.info "Dry run: would have executed hook: system(#{file}, #{failover_ip}, #{old_ip}, #{new_ip})"
+        end
```

Comment:
could you change the if/else here and in failover_ip.rb to non-negative style

```
if dry
  # just log stuff
else
  # do the magic
end
```

Reply:
grazie! -> 266c4dc